app/src/main/java/org/fossasia/susi/ai/activities/LoginActivity.java

```diff
@@ -112,6 +119,15 @@ public void forgotPassword() {
         startActivity(intent);
     }
 
+    @OnClick(R.id.skip)
+    public void skip() {
+        Intent intent = new Intent(LoginActivity.this,MainActivity.class);
+        intent.putExtra(getApplicationContext().getString(R.string.first_time), false);
```

Comment:
Use Contant.FIRST_TIME instead of getApplicationContext().getString(R.string.first_time).